### DISCUSSION TITLE
IMAGE TAG

### DISCUSSION THREAD

**Discussion by madannaik** [vercel/next.js#29858] [Help]
### Describe the feature you'd like to request

Dude, I need a normal img tag in NEXT js . Please consider my request.

### Describe the solution you'd like

I want to use a normal HTML img tag in code .

### Describe alternatives you've considered

I am really irritated with the Image tag in NEXT js and I have moved to create-next-app again. Please make it happen I want to use next js again as it is simpler to use than normal create-react-app code.

**Comment by MonstraG** [comment]
Just so you know, nobody forbids you from using <img> tag.

If you're running eslint, you can disable the rule:

.eslintrc
```
{
	"rules": {
		"@next/next/no-img-element": "off"
	}
}
```
